radis/extractions/tests/unit/test_llm_extraction.py

```
@@ -0,0 +1,293 @@
+
+    assert set(fields) == {"finding", "size", "malignant"}
+    assert fields["finding"].annotation is str
+    assert fields["malignant"].annotation is bool
+    # NUMERIC maps to ``float | int`` (a Union), so a float and an int validate.
+    # Schema is built dynamically, so its fields aren't statically known to the type
+    # checker; access them through Any-typed instances.
+    instance = cast(Any, Schema(finding="x", size=3.5, malignant=True))
+    assert instance.size == 3.5
+    assert cast(Any, Schema(finding="x", size=7, malignant=False)).size == 7
+    # All fields are required (``...``); omitting one is a validation error.
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)